erp/management/commands/export_to_datagouv.py

```diff
@@ -22,18 +22,28 @@ def add_arguments(self, parser):
             help="Afficher les infos",
         )
 
+        parser.add_argument(
+            "--local",
+            action="store_true",
+            help="Export local uniquement",
+        )
+
     def log(self, msg):
         if self.verbose:
             print(msg)
 
     def handle(self, *args, **options):
-        # self.verbose = options.get("verbose", False)
+        self.verbose = options.get("verbose", False)
+        local = options.get("local", False)
+
         csv_path = "acceslibre.csv"
         self.log("Starting export")
         try:
             erps = Erp.objects.published().having_a11y_data()
             export_schema_to_csv(csv_path, erps, EtalabMapper)
             self.log("Local export successful: 'acceslibre.csv'")
+            if local:
+                return
```

Comment:
How about a more classic `--dry-run` option?

Reply:
I want to run it and create the export, but not to upload it. It helps me to update the samples data required for testing. Maybe a `--no-upload` or `--local-only` flag maybe more expressive ?

Reply:
`--skip--upload` sounds good